templates/files/empty-server/server/middleware.development.json

```diff
@@ -1,10 +0,0 @@
-{
-  "final:after": {
```

Comment:
IIUC, you are changing the template of `empty-server` here, compared to what the current loopback-workspace@3 scaffolds.

What's the reasoning behind this change?